editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4090,6 +4090,8 @@ void CanvasItemEditor::_notification(int p_what) {
 		anchors_popup->add_icon_item(get_theme_icon("ControlAlignWide", "EditorIcons"), TTR("Full Rect"), ANCHORS_PRESET_WIDE);
 
 		anchor_mode_button->set_icon(get_theme_icon("Anchor", "EditorIcons"));
+
+		zoom_reset->add_theme_color_override("font_color", Color(1, 1, 1));
```

Comment:
Ah now that I think about it, if it's meant to *always* be white and not dynamic, then it's fine to change it in the constructor instead of here. Changing in `NOTIFICATION_EDITOR_SETTINGS_CHANGED` is for stuff that depends on the theme and thus needs to be redone each time you change theme.

That being said, I guess this only works as long as you use a dark default clear color in the project settings. If you set a light color as clear color, you'd get the same issue again.

Reply:
Maybe we could make the font outline for this label thicker, so it's visible on any color?

Reply:
Huum, that is true, I didn't remember the "clear color" property. Then it's better to implement a solution similar to #36530 and draw a semi-opaque box behind.

Reply:
How about something like #36530? (we could use the `_draw_text_with_backdrop` method if that was merged)

Reply:
@KoBeWi I though about it, but `zoom_reset` is a Button, so it wouldn't work. On the other hand *(because it is a Button)*, we can just re-enable it's background panel. However adding an outline to the text also works pretty well, and is less distracting *(however it might not be very visible in some cases)*

Here's the text with outline, and with the button background, for comparision.

![image](https://user-images.githubusercontent.com/6501975/96905949-0d3f5580-1491-11eb-803f-16ba6602ecfc.png)

Which one do you guys think looks best?

Reply:
@Ev1lbl0w The second one, which I have been suggesting from the beginning.